src/fmu/dataio/export/rms/structure_depth_isochores.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, Final
+
+import fmu.dataio as dio
+from fmu.dataio._logging import null_logger
+from fmu.dataio._models.fmu_results import standard_result
+from fmu.dataio._models.fmu_results.enums import Classification, StandardResultName
+from fmu.dataio.exceptions import ValidationError
+from fmu.dataio.export._decorators import experimental
+from fmu.dataio.export._export_result import ExportResult, ExportResultItem
+from fmu.dataio.export.rms._utils import (
+    get_rms_project_units,
+    get_zones_in_folder,
+    load_global_config,
+)
+
+if TYPE_CHECKING:
+    import xtgeo
+
+_logger: Final = null_logger(__name__)
+
+
+class _ExportStructureDepthIsochores:
+    def __init__(
+        self,
+        project: Any,
+        horizon_folder: str,
+    ) -> None:
+        _logger.debug("Process data, establish state prior to export.")
+        self._config = load_global_config()
+        self._surfaces = get_zones_in_folder(project, horizon_folder)
+        self._unit = "m" if get_rms_project_units(project) == "metric" else "ft"
+
+        _logger.debug("Process data... DONE")
+
+    @property
+    def _standard_result(self) -> standard_result.StructureDepthIsochoreStandardResult:
+        """Standard result type for the exported data."""
+        return standard_result.StructureDepthIsochoreStandardResult(
+            name=StandardResultName.structure_depth_isochore
+        )
+
+    @property
+    def _classification(self) -> Classification:
+        """Get default classification."""
+        return Classification.internal
+
+    @property
+    def _subfolder(self) -> str:
+        """Subfolder for exporting the data to."""
+        return "structure_depth_isochores"
```

Comment:
I would like to do this instead, but it feels like the folder name should be plural 😄 .. but does it really matter? What do you think @mferrera ? Would be nice to have this consistent across the standard results, but we used structure_depth_surface **+s**  for the `structure_depth_surface` standard result.
```suggestion
        return self._standard_result.value
```

Reply:
hehe yes, maybe.. feels a bit much just to have some plurality in the folder name 😄  I think I lean towards the folder name being identical to the standard result name for all - dropping the "s" even though multiple items are exported. Would be a clean pattern.